docs/02-Install.md

```diff
@@ -8,6 +8,202 @@ Dependencies and installation information are documented on the LORIS-MRI
+    the Apache linux user.
+- the Configuration module (*Paths*) `Imaging data`, `MINC files` and 
+  `Images` settings are set (typically: `/data/$PROJECT/data/`). 
+    
+More detailed specifications can be consulted in the 
+[LORIS repository: Imaging Browser Specification](https://github.com/aces/Loris/blob/master/modules/imaging_browser/README.md).
+
+4. **Brainbrowser**
+
+Brainbrowser displays the MINC images within the browser. It accesses those MINC 
+images directly from the filesystem. Ensure that:
+- `/data/$PROJ` directory and subdirectories are readable and executable by
+    the Apache linux user.
+- the Configuration module (*Paths*) `MINC files` setting is
+    `/data/$PROJ/data/`.
+- the _project/config.xml_ file (in the main LORIS codebase) contains the
+      proper MINC toolkit path in the `<MINCToolsPath>` tagset.
+      
+More detailed specifications can be consulted in the 
+[LORIS repository:Brainbrowser Specification](https://github.com/aces/Loris/blob/master/modules/brainbrowser/README.md).
+
+
+5. **MRI Violated Scans**
+
+No configuration setting is needed for the MRI Violated Scans module to work. 
+Data loaded in this module gets populated automatically by the insertion 
+scripts. As such, scans whose parameters can't be matched against the 
+`mri_protocol` table during the imaging insertion process, will be flagged as 
+protocol violations and will not have their MINC/NIfTI volumes loaded in the 
+database. The type of error (scan identification, protocol violation) will be 
+listed and can be reviewed from the front-end.
+
+More detailed specifications can be consulted in the 
+[LORIS repository:MRI Violated Scans Specification](https://github.com/aces/Loris/blob/master/modules/mri_violations/README.md).
+
+
+### 2.2.3 LORIS-MRI 
+
+#### Filesystem
+
+- `/data/*` subdirectories were created by the imaging install script. If not,
+    it may be due to `root:root` ownership of the `/data/` mount on your
+    system. Ensure these subdirectories are created manually, particularly:
+    `/data/$PROJ/data/*`, `/data/$PROJ/bin/mri/` and `/data/incoming/`
+
+- `/data/$PROJ/` directory and subdirectories must be readable and executable
+    by the Apache linux user. It may also help to ensure the `/data/` mount is
+    executable. After any modifications, ensure you restart apache.
+    
+#### Customizable routines in the `prod` file
+
+- `isFileToBeRegisteredGivenProtocol()`
+
+    * By default, any scan will be inserted if it matches an _mri_protocol_ 
+    table entry.
+    * To **whitelist/blacklist** specific scan types -- e.g. in the case of 
+    protocol exclusion, case sensitivity or labelling variance -- modify the 
+    subroutine, e.g.:
+
+```perl
+if($acquisitionProtocol eq 't1' or $acquisitionProtocol eq 't2' or $acquisitionProtocol eq 'dti' or $acquisitionProtocol eq 'bold' or $acquisitionProtocol =~ /fmri/) { return 1; }
+```
+
+- getSNRModalities()
```

Comment:
getSNRModalities() should be in code format (applicable to all other functions mentioned below)

Reply:
good catch thanks